debian/ubuntu-pro-client.postinst

```diff
@@ -40,6 +40,30 @@ mark_reboot_cmds_as_needed() {
       touch $REBOOT_CMD_MARKER_FILE
     fi
 }
+create_onlyseries_marker_file() {
+    /usr/bin/python3 -c "
+from uaclient.files import MachineTokenFile, state_files
+machine_token_file = MachineTokenFile()
+content = machine_token_file.read()
+contractInfo = content.get("machineTokenInfo", {}).get(
+    "contractInfo", {}
+)
+support_resource = dict(
+    (e.get("type"), e)
+    for e in contractInfo.get("resourceEntitlements", [])
+    if e.get("type") == "support"
+)
+only_series = (
+    support_resource.get("support", {})
+    .get("affordances", {})
+    .get("onlySeries", None)
+)
+if only_series:
+    state_files.only_series_check_marker.write(only_series)
+"
+}
+
+create_onlyseries_marker_file
```

Comment:
@orndorffgrant @lucasmoura I think there is a better way to handle this in the postinst, please feel free to leave suggestions for improvement. Thanks!

Reply:
Aside: this needs to go into the postinst-migrations.sh file

My initial thoughts are:

I think for postinst it would be better to not rely on our MachineTokenFile abstraction. Instead, check if the machine token is there in `sh`. If it is, go into python but read it with the normal python `json` module.

Also finding the support dict can be a bit simpler:
```python
ents = dict.get("machineTokenInfo", {}).get("contractInfo", {}).get("resourceEntitlements", [])
support = next([e for e in ents if e.get("type") == "support"], {})
```
then the python script should just print the value of onlyseries and the `sh` can check if it is nonempty and `touch` the marker file.

Reply:
Updated @orndorffgrant, does this look better?